src/context.jl

```diff
@@ -38,6 +38,14 @@ function Context(f::Function, args...)
     end
 end
 
+function Base.show(io::IO, ctx::Context)
+    if isopen(ctx)
+        print(io, Context, "($(getfield(ctx, :sockets)))")
+    else
+        print(io, Context, "([closed])")
```

Comment:
This looks a little too much like an array with one element `closed`…

Reply:
I copied it from a Base object but I don't recall which one now :thinking: What do you think about `Context() (closed)` in a95c73b? That copies the style of `Channel`.

Reply:
(bump)